### DISCUSSION TITLE
Google以第四代TPU加速器建構最大規模的機器學習中樞 以90%無碳排放能源驅動

### DISCUSSION THREAD

**Discussion by timchen1216** [1101-GS3073-C/Python-Community-GS3073-C#367] [科技新知]
![image](https://user-images.githubusercontent.com/95116299/172308525-d37c5207-9651-4439-b96a-a51aae31cf3d.png)
去年宣布推出第四代TPU加速器之後，Google在此次Google I/O 2022宣布以8組Cloud TPU v4 Pod運算裝置構成最大規模的[機器學習](https://udn.com/search/tagging/2/%E6%A9%9F%E5%99%A8%E5%AD%B8%E7%BF%92)中樞，更強調使用90%無碳排放能源驅動。

在先前說明中，Google表示第四代TPU加速器運算效能相比前一代產品提昇一倍，同時藉由4096組第四代TPU加速器構成的POD運算裝置，更可對應1 exaFLOPS以上的運算表現 (即每秒1018次浮點運算)。而此次以8組Cloud TPU v4 Pod運算裝置構成的機器學習中樞，在峰值運算最高可達每秒900萬兆次浮點運算，成為全球規模最大的公開機器學習中樞，並且將透過Google Cloud服務對外開放使用。

包含Cohere、LG人工智慧研發中心 (LG [AI](https://udn.com/search/tagging/2/AI) Research)、Meta AI，以及Salesforce Research等業界團隊都已經實際體驗Google此機器學習中樞，並且可透過TPU VM架構自行設定互動式開發環境，並且能彈性應用JAX、PyTorch，或是TensorFlow等機器學習框架，另外也藉由快速互聯與最佳化軟體堆棧特性，藉此打造更突出效能與擴充性表現。

而Google此次在Google I/O 2022提及的PaLM (Pathways Language Model)[語言](https://udn.com/search/tagging/2/%E8%AA%9E%E8%A8%80)模型，便是透過2組TPU v4 Pod訓練而成，藉此對應更迅速的語言翻譯、理解效果。

資料來源:https://udn.com/news/story/7086/6306929

我好想要一顆TPUỎ̷͖͈̞̩͎̻̫̫̜͉̠̫͕̭̭̫̫̹̗̹͈̼̠̖͍͚̥͈̮̼͕̠̤̯̻̥̬̗̼̳̤̳̬̪̹͚̞̼̠͕̼̠̦͚̫͔̯̹͉͉̘͎͕̼̣̝͙̱̟̹̩̟̳̦̭͉̮̖̭̣̣̞̙̗̜̺̭̻̥͚͙̝̦̲̱͉͖͉̰̦͎̫̣̼͎͍̠̮͓̹̹͉̤̰̗̙͕͇͔̱͕̭͈̳̗̭͔̘̖̺̮̜̠͖̘͓̳͕̟̠̱̫̤͓͔̘̰̲͙͍͇̙͎̣̼̗̖͙̯͉̠̟͈͍͕̪͓̝̩̦̖̹̼̠̘̮͚̟͉̺̜͍͓̯̳̱̻͕̣̳͉̻̭̭̱͍̪̩̭̺͕̺̼̥̪͖̦̟͎̻̰_Ỏ̷͖͈̞̩͎̻̫̫̜͉̠̫͕̭̭̫̫̹̗̹͈̼̠̖͍͚̥͈̮̼͕̠̤̯̻̥̬̗̼̳̤̳̬̪̹͚̞̼̠͕̼̠̦͚̫͔̯̹͉͉̘͎͕̼̣̝͙̱̟̹̩̟̳̦̭͉̮̖̭̣̣̞̙̗̜̺̭̻̥͚͙̝̦̲̱͉͖͉̰̦͎̫̣̼͎͍̠̮͓̹̹͉̤̰̗̙͕͇͔̱͕̭͈̳̗̭͔̘̖̺̮̜̠͖̘͓̳͕̟̠̱̫̤͓͔̘̰̲͙͍͇̙͎̣̼̗̖͙̯͉̠̟͈͍͕̪͓̝̩̦̖̹̼̠̘̮͚̟͉̺̜͍͓̯̳̱̻͕̣̳͉̻̭̭̱͍̪̩̭̺͕̺̼̥̪͖̦̟͎̻̰

**Comment by Joan0518** [comment]
這是什麼酷酷的顏文字哈哈哈
<img width="194" alt="擷取" src="https://user-images.githubusercontent.com/90574174/172339103-48726aa5-b40a-44ce-8ccf-f341b6442a2d.PNG">

**Comment by Bill671** [comment]
哈哈，也太酷 Ỏ̷͖͈̞̩͎̻̫̫̜͉̠̫͕̭̭̫̫̹̗̹͈̼̠̖͍͚̥͈̮̼͕̠̤̯̻̥̬̗̼̳̤̳̬̪̹͚̞̼̠͕̼̠̦͚̫͔̯̹͉͉̘͎͕̼̣̝͙̱̟̹̩̟̳̦̭͉̮̖̭̣̣̞̙̗̜̺̭̻̥͚͙̝̦̲̱͉͖͉̰̦͎̫̣̼͎͍̠̮͓̹̹͉̤̰̗̙͕͇͔̱͕̭͈̳̗̭͔̘̖̺̮̜̠͖̘͓̳͕̟̠̱̫̤͓͔̘̰̲͙͍͇̙͎̣̼̗̖͙̯͉̠̟͈͍͕̪͓̝̩̦̖̹̼̠̘̮͚̟͉̺̜͍͓̯̳̱̻͕̣̳͉̻̭̭̱͍̪̩̭̺͕̺̼̥̪͖̦̟͎̻̰_Ỏ̷͖͈̞̩͎̻̫̫̜͉̠̫͕̭̭̫̫̹̗̹͈̼̠̖͍͚̥͈̮̼͕̠̤̯̻̥̬̗̼̳̤̳̬̪̹͚̞̼̠͕̼̠̦͚̫͔̯̹͉͉̘͎͕̼̣̝͙̱̟̹̩̟̳̦̭͉̮̖̭̣̣̞̙̗̜̺̭̻̥͚͙̝̦̲̱͉͖͉̰̦͎̫̣̼͎͍̠̮͓̹̹͉̤̰̗̙͕͇͔̱͕̭͈̳̗̭͔̘̖̺̮̜̠͖̘͓̳͕̟̠̱̫̤͓͔̘̰̲͙͍͇̙͎̣̼̗̖͙̯͉̠̟͈͍͕̪͓̝̩̦̖̹̼̠̘̮͚̟͉̺̜͍͓̯̳̱̻͕̣̳͉̻̭̭̱͍̪̩̭̺͕̺̼̥̪͖̦̟͎̻̰ ，這表情符號在哪裡找的，給我來一打

**Comment by luo910227** [comment]
我認為GOOGLE能同時兼顧科技發展與環境保護是可貴的，標榜使用90%無碳排放能源驅動也勢必能大大增加使用者支持度
